src/plots/cartesian/layout_attributes.js

```diff
@@ -304,6 +304,19 @@ module.exports = {
             'the axis lines.'
         ].join(' ')
     },
+    tickson: {
+        valType: 'enumerated',
+        values: ['labels', 'boundaries'],
+        role: 'info',
+        editType: 'ticks',
```

Comment:
dflt?

Reply:
Good call. Done in -> https://github.com/plotly/plotly.js/pull/3275/commits/78ef22ce125429ae0339b355069ad29f6ce60dc6